From b89993aa784e9c80a4ea7193d96f070acff8ab00 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sat, 14 May 2022 22:57:37 +0800
Subject: [PATCH 20/58] Revert "drm/msm/dsi: Use connector directly in
 msm_dsi_manager_connector_init()"

This reverts commit 98a7f6c4ada4ca97fcb0d508e097f8597df19d4d.
---
 drivers/gpu/drm/msm/dsi/dsi_manager.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/msm/dsi/dsi_manager.c b/drivers/gpu/drm/msm/dsi/dsi_manager.c
index fb421ca56b3da..1d28dfba2c9bb 100644
--- a/drivers/gpu/drm/msm/dsi/dsi_manager.c
+++ b/drivers/gpu/drm/msm/dsi/dsi_manager.c
@@ -644,7 +644,7 @@ struct drm_connector *msm_dsi_manager_connector_init(u8 id)
 	return connector;
 
 fail:
-	connector->funcs->destroy(connector);
+	connector->funcs->destroy(msm_dsi->connector);
 	return ERR_PTR(ret);
 }
 
-- 
2.27.0

